### ISSUE TITLE
Dataset

### ISSUE THREAD

**Issue by Daybreak-Zheng** [Johnathan-Xie/ZSD-YOLO#1]
Hello, excellent work. May I ask how the label file in PT format in the training set is generated? Looking forward to your reply

**Comment by Johnathan-Xie** [comment]
Hi, these label files are generated when running torch.save() on the tensor which contains the class, bounding box, and embedding vectors. All data processing was performed via notebooks which I could share if you need to process a new dataset.
